build.gradle

```diff
@@ -90,13 +90,13 @@ dependencies {
     def junitPlatform = "1.9.3"
     def jaxbVersion = "2.3.1"
 
-    implementation("org.opensearch:opensearch:${opensearchVersion}")
+    api("org.opensearch:opensearch:${opensearchVersion}")
```

Comment:
`api` tends to make the internal libraries as public whereas `implementation` won't let us access the internal libraries by considering them `private`. I don't understand the reason for this change.

Reply:
Okay. I see #609